de.dashup.application/src/main/webapp/views/settings.jsp

```diff
@@ -94,11 +94,11 @@
                                 </div>
                                 <div class="input-field col s8 m8">
                                     <select name="theme" id="theme-dropdown">
-                                        <option value="blue-sky">Blue Sky</option>
-                                        <option value="green-nature">Green Nature</option>
-                                        <option value="red-love">Red Love</option>
-                                        <option value="white-diamond">White Diamond</option>
-                                        <option value="black-night">Black Night</option>
+                                        <option value="Blue Sky">Blue Sky</option>
```

Comment:
Why removing the technical name? This technical name is unique and equal to the theme's style sheet.

Reply:
I did not know why you introduced the technical name until I found out that you bind the stylesheet name to the model, which uses the technical name of course as whitespace is not allowed. Then I rolled it back^^